packages/utils/src/isFullPath.ts

```diff
@@ -0,0 +1,13 @@
+export const isFullPath = (path: string) => {
+    // Full Unix-like and Windows paths
+    const fullPathPattern = /^(\/|([a-zA-Z]:\\))/;
+
+    // Check to avoid matching URLs
+    const urlPattern = /^(https?:\/\/|www\.)/;
+
+    // Check for full path patterns, excluding URLs
+    const isFullPath = fullPathPattern.test(path);
```

Comment:
Eslint error:
```'isFullPath' is already declared in the upper scope on line 1 column 14.```

Reply:
Let's simplify it like https://github.com/trezor/trezor-suite/commit/6a40e46a6c52c8df049c4d8d45434be4adc8651a